xAPI.md

```diff
@@ -3226,10 +3226,16 @@ include attachment raw data and MUST report application/json.
 
 ###### Filter Conditions for StatementRefs 
 
-For filter parameters which are not time or sequence based (that is, other than
-since, until, or limit), Statements which target another Statement (by using a StatementRef
+This section outlines rules by which additional Statements are considered to meet the filter coniditions even
+if they do not match the original query's filter parameters but instead target a Statement which does. 
```

Comment:
For this sentence to be correct, we have to introduce a second meaning of matching the filter parameters below (instead of just talking about matching some of the filter parameters as before). That's a lot of extra complication.